job_offers_data_load/update_MongoDB_procedure.py

```diff
@@ -0,0 +1,48 @@
+import pandas
+from job_offers_data_load import MongoDB_connection_functions
+import datetime
+import pymongo
+
+
+def check_update_or_create_MongoDB(new_data: pandas.DataFrame,
+                                   host: str, port: str,
+                                   db_name: str, collection_name: str) -> None:
+    """Checks if collection has any records, if not calls create_MongoDB, if yes calls update_MongoDB."""
+    #  create connection with MongoDB
+    collection = MongoDB_connection_functions.connection_to_mongodb(host, port, db_name, collection_name)
+    if collection.estimated_document_count() == 0:
+        create_MongoDB(new_data, collection)
+    elif collection.estimated_document_count() > 0:
+        update_MongoDB(new_data, collection)
+    else:
+        print("Cannot find existing collection nor create one.")
+
+
+def create_MongoDB(new_data: pandas.DataFrame, collection: pymongo.collection.Collection) -> None:
+    """Creates MongoDB database by inserting pandas DataFrame into it."""
+    for row in new_data.to_dict('records'):
+        collection.insert_one(row)
+    print("Finished creating MongoDB.")
+
+
+def update_MongoDB(new_data: pandas.DataFrame, collection: pymongo.collection.Collection) -> None:
+    """Updates existing database. Checks currently existing records, decides if they already expired or not
+    and stamps accordingly."""
+    #  get data from DB to later compare it with newly scraped data
+    old_data = pandas.DataFrame(list(collection.find()))
```

Comment:
Load data as python dicts.
Maybe convert into custom classes (you can use https://docs.python.org/3/library/dataclasses.html).